cypress/features/regression/advancedColorPicker.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Advanced Color Picker component
+  I want to test Advanced Color Picker component
+
+  Background: I open basic Test Advanced Color Picker component page
+    When I open basic Test "Advanced Color Picker" component page
+      And I open Advanced Color Picker
+
+  @positive
+  Scenario: Tab key pressed two times when color is focused regains focus on color
```

Comment:
too many unnecessary steps. You should have only two steps like press tab two times and simple color has focus

Reply:
reduces amount of steps. done 👍